safe/impact_function/impact_function.py

```diff
@@ -2933,4 +2935,88 @@ def generate_report(self, components, output_folder=None, iface=None):
         if map_overview_layer:
             QgsMapLayerRegistry.instance().removeMapLayer(map_overview_layer)
 
+        # Create json file for report urls
+        report_path = dirname(self._analysis_impacted.source())
+        filename = join(report_path, 'output', 'report_metadata.json')
+        write_json(self.report_urls(), filename)
```

Comment:
ah sorry, I missed this one. Ignore my other comment

Reply:
trolled